api/pyproject.toml

```diff
@@ -38,6 +38,7 @@ matplotlib = "^3.2.2"
 pytest-xdist = "^1.33.0"
 rope = "^0.17.0"
 docker-compose = "^1.26.2"
+pgadmin4 = "^4.25"
```

Comment:
What do we use pgadmin for?

Reply:
pgadmin is the postgresql ui. not sure if it's the same team who maintains it, I think it used to be. It doesn't HAVE to be in as a dependancy, and we could drop it. It's easy enough to just pip install it local.
Now that I think about it -  I'm going to drop it. That and docker-compose - it's just be making the automated testing slower. I can pip install it local and it's fine.

Reply:
Oh wait - never mind - we're not doing a --dev so it's fine - never mind.

This is only happening on our local machines, unless I'm missing something, so it's fine.